main.tf

```diff
@@ -34,6 +34,10 @@ resource "aws_lambda_function" "this" {
   tags                           = var.tags
   timeout                        = var.timeout
 
+  dead_letter_config {
```

Comment:
@3h4x looks like you'll need to make this block dynamic based on the value of the variable. Tests are hitting the following: 

```
Error: Missing required argument

  on ../../main.tf line 38, in resource "aws_lambda_function" "this":
  38:     target_arn = var.dead_letter_config_target_arn

The argument "dead_letter_config.0.target_arn" is required, but no definition
was found.
}
    apply.go:15: 
        	Error Trace:	apply.go:15
        	            				examples_complete_test.go:37
        	Error:      	Received unexpected error:
        	            	FatalError{Underlying: error while running command: exit status 1; 
        	            	Error: Missing required argument
        	            	
        	            	  on ../../main.tf line 38, in resource "aws_lambda_function" "this":
        	            	  38:     target_arn = var.dead_letter_config_target_arn
        	            	
        	            	The argument "dead_letter_config.0.target_arn" is required, but no definition
        	            	was found.
        	            	}
        	Test:       	TestExamplesComplete
```

Mind doing that update?

Reply:
I added and committed the above change